packages/types/src/sth-rest-api/data-transfer-objects/sequence-dto.ts

```diff
@@ -0,0 +1,7 @@
+import { SequenceConfig } from "../../runner-config";
+
+export type SequenceDTO = {
```

Comment:
Ok, we're either using DTOs everywhere or nowhere. It's a sensible approach, but should be done in a more thought-through way.

Reply:
yeah, it's sth I started to make a distinction between Sequence type we use internally and the returned data from REST API. I guess we don't have a layer of data transfer right now, it's a bit all over the place :wink:

Reply:
removed it :)